slides/src/content/component.md

```diff
@@ -226,13 +228,13 @@ export class Child {
 
 ## Quiz (continued)
 
-1. Parent:  
+1. Parent:
```

Comment:
The two spaces at the end of a line force a line break in Markdown - without them, the formatting here (and below) will be off. Philip uses an explicit `<br>` to make the line break visible and to prevent editors stripping it out - can you do that here?